server/src/option.rs

```diff
@@ -351,6 +356,24 @@ impl Server {
                     .required(true)
                     .help("Password for the basic authentication on the server"),
             )
+            .arg(
+                Arg::new(Self::SEND_ANALYTICS)
+                    .long(Self::SEND_ANALYTICS)
+                    .env("P_SEND_ANONYMOUS_USAGE_DATA")
+                    .value_name("BOOL")
+                    .required(false)
+                    .default_value("true")
+                    .value_parser(value_parser!(bool))
+                    .help("Disable/Enable sending anonymous user data"),
+            )
+            .arg(
+                Arg::new(Self::OPEN_AI_KEY)
+                    .long(Self::OPEN_AI_KEY)
+                    .env("OPENAI_API_KEY")
```

Comment:
This should be `P_OPENAI_API_KEY`, so we're consistent with the format we've used till now.

Reply:
Resolved in https://github.com/parseablehq/parseable/pull/492 👍